update_configs.py

```diff
@@ -0,0 +1,25 @@
+"""
+Update configurations for VSCode and other editors.
+VSCode at least does not support composing configurations (https://github.com/microsoft/vscode/issues/15909)
+and we don't want to clobber any existing configurations.
+"""
+import json
+from pathlib import Path
+from typing import Any, Dict
+
+VSCODE_CONFIG_DIR = Path('.vscode')
+VSCODE_CONFIG_DIR.mkdir(exist_ok=True)
```

Comment:
This will create .vscode folder on everyone's machine (even non-users). Does it make sense to just exit early if it doesn't already exist? That assumes you'd run `make install` after having opened vscode. I'm okay either way, not sure if @samuelcolvin cares. But @adriangb would you care if we created a `.idea`/similar folder for you?

Reply:
I think we can just create it, it's not a big deal. I wouldn't care the other way around.

Reply:
Humm, I'd prefer to only create the file if we somehow know they're using vscode.